modules/spark/src/main/scala/org/apache/ignite/spark/impl/package.scala

```diff
@@ -71,29 +74,22 @@ package object impl {
     def sqlTableExists(ignite: Ignite, tabName: String, schemaName: Option[String]): Boolean =
         sqlTableInfo(ignite, tabName, schemaName).isDefined
 
-    /**
-      * @param ignite Ignite instance.
-      * @param tabName Table name.
-      * @param schemaName Optional schema name.
-      * @return QueryEntity for a given table.
-      */
-    def igniteSQLTable(ignite: Ignite, tabName: String, schemaName: Option[String]): Option[QueryEntity] =
-        sqlTableInfo[Any, Any](ignite, tabName, schemaName).map(_._2)
-
     /**
       * @param ignite Ignite instance.
       * @param tabName Table name.
       * @param schemaName Optional schema name.
       * @return Cache name for given table.
       */
     def sqlCacheName(ignite: Ignite, tabName: String, schemaName: Option[String]): Option[String] =
-        sqlTableInfo[Any, Any](ignite, tabName, schemaName).map(_._1.getName)
+        if (sqlTableInfo(ignite, tabName, schemaName).isDefined)
```

Comment:
Let's use `map` here, like in previous implementation.

Reply:
Fixed